### PR TITLE
SPCTR-1750 Fix: Advanced Columns - Columns reverse functionality not working for old user.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

https://brainstormforce.atlassian.net/browse/SPCTR-1750

### Screenshots

- NA

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/columns/style.scss [hunk 1]
```diff
@@ -201,12 +201,17 @@
 		}
 	}
 
-	.uagb-columns__stack-tablet.uagb-columns__reverse-tablet {
+	.uagb-columns__stack-tablet.uagb-columns__reverse-tablet .uagb-columns__inner-wrap {
 		display: flex;
 		flex-direction: column-reverse;
```
Comment (RajkiranBagal) [inline src/blocks/columns/style.scss:206]: Please make good use of SCSS, Also we can merge these CSS in one too

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/columns/style.scss (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/columns/style.scss]: Done

File: src/blocks/columns/style.scss [hunk 2]
```diff
@@ -229,14 +230,24 @@
 		> .uagb-columns__inner-wrap { // For Backword.
 			display: block;
 		}
+
+		&.uagb-columns__reverse-mobile .uagb-columns__inner-wrap {
+			display: flex;
+			flex-direction: column-reverse;
```
Comment (RajkiranBagal) [inline src/blocks/columns/style.scss:236]: Both these CSS can be combined

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/columns/style.scss (hunk 2 above)
Comment (AbhijitNage123) [reply src/blocks/columns/style.scss]: Done

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**